tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -130,6 +130,19 @@ def test_name_from_100__a_b_u():
 
     assert expected == result['name']
 
+def test_name_from_100__b_u_no_a():
+    snippet = (
+        '<datafield tag="110" ind1=" " ind2=" ">'
+        '  <subfield code="b">Department of Physics</subfield>'
+        '  <subfield code="u">Fukushima U.</subfield>'
+        '</datafield>'
+    )  # record/902812
```

Comment:
Wait a second! This comment is not cosmetic: the record you point to should have this behaviour, to show that this is a bug you actually encountered on real, production data. Like these: https://inspirehep.net/search?ln=en&cc=Institutions&ln=en&cc=Institutions&p=110__u%3A**+and+110__b%3A**+and+not+110__a%3A**&action_search=Search&sf=institutionname&so=a&rm=&rg=25&sc=0&of=hb